Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/API.hs

```diff
@@ -155,6 +157,8 @@ makeBookingAPIEntity booking activeRide allRides fareBreakups mbExophone mbPayme
       paymentUrl = booking.paymentUrl,
       createdAt = booking.createdAt,
       updatedAt = booking.updatedAt,
+      startOdometerReading = activeRide >>= (.startOdometerReading),
```

Comment:
This data should be inside `rideList` field, not here.

Reply:
We do not have `rideList` at customer side